src/inverseRobot.js

```diff
@@ -23,8 +23,24 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  let swappedObject = [];
+  const mySet = new Set();
+  const arrRobot = Object.values(robot);
```

Comment:
you can get values and create a set of unique values
```suggestion
  const values = Object.values(robot);
  const uniqueValues = new Set(values).size;
```